database/postgres/event.go

```diff
@@ -24,8 +24,8 @@ const (
 	createEvent = `
 	INSERT INTO convoy.events (id,event_type,endpoints,project_id,
 	                           source_id,headers,raw,data,url_query_params,
-	                           idempotency_key,is_duplicate_event,created_at,updated_at)
-	VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12, $13)
+	                           idempotency_key,is_duplicate_event,acknowledged_at)
```

Comment:
I'm a bit concerned about this, i agree it's better to use `acknowledged_at` at the entry point when we queue the event helps to record latency correctly, but as at the point when we insert it into the db, isn't now in the true sense, created? By this logic I think `created_at,updated_at` should still be inserted here.

Reply:
The db should tell when it is created. At this point, it is source of truth.

Reply:
@mekilis i don't understand, are we not saying the same thing?